### DISCUSSION TITLE
Has there been an attempt to use other spiking neuron besides LIF neuron?

### DISCUSSION THREAD

**Discussion by AreUTheDoctor** [jeshraghian/snntorch#97] [Q&A]
Just wonder how Izhikevich neuron would perform in snnTorch for example

**Comment by jeshraghian** [comment]
@vinniesun has been playing around with Izhikevich neuron models, have you seen any improvement?

From what I've read so far, a few papers that have trained networks of Izhikevich neurons using backprop haven't yet appeared to be competitive against leaky integrate-and-fire neurons. While adding this neuron is on the to-do list, it's been a low priority.

As for other neurons, I have included Spiking LSTMs (less biological, more equipped for deep learning) and several other LIF variants. Full list is here: https://snntorch.readthedocs.io/en/latest/snntorch.html

**Reply by AreUTheDoctor** [reply]
Since you have mentioned LSTM, how does STDP fair against it for deep learning?
